src/lib/nvme_pt_ioctl.c

```diff
@@ -723,14 +723,15 @@ static int opal_start_anybody_session(int fd, struct opal_device *dev, int sp)
 		dev->session.tsn = dev->payload.tokens[5]->vals.uint;
```

Comment:
can we make this a little bit pretty?
ret = opal_snd_rcv_cmd_parse_chk(fd, dev, false);
if (ret != OPAL_SUCCESS) {
    SEDCLI_DEBUG_MSG("Response parsed incorrectly.\n");
   goto put_tokens;
}
dev->session.hsn = dev->payload.tokens[4]->vals.uint;
dev->session.tsn = dev->payload.tokens[5]->vals.uint;



}